### ISSUE TITLE
Add option to center contents with limited text-width

### ISSUE THREAD

**Issue by daniel-dietrich** [newsboat/newsboat#1320]
I believe one of the main selling points of feed readers is their distraction-free nature and comfort of use. In my experience, having text centered with narrow width limit in maximized window is the most enjoyable way to read. Currently I was able to only limit the text width.

**Comment by Minoru** [comment]
Hi, thanks for opening the issue!

I think the option makes sense. I'll accept a pull request adding it.

A few thoughts:
* if `html-renderer` is set, `text-width` doesn't apply — the new setting probably shouldn't apply either;
* we should make sure that the text stays centred even when the window is resized.

**Comment by Mati20187** [comment]
Hi, it's quite an old issue so I wanted to ask whether it's been implemented yet. 
Unfortunately, I don't have any programming skills but ever since I started using miniflux, web client on mobile (which centers text) and newsboat as a client on desktop, I felt that it's an important feature that many users would appreciate

**Comment by Minoru** [comment]
@Mati20187 Nope, it hasn't been implemented yet. We don't do any sort of planning, so I can't tell you when it will get done — it will be implemented when someone decides to implement it and send a pull request.

**Comment by Friptick** [comment]
This has been bothering me too. On a wider screen it just looks odd to have the column "stuck" to one side. Especially coming from a decent terminal epub reader like `epy`, where the column is where it might be expected: the middle.

Pity about the PR above. What about a basic solution `left-margin x` which just shifts everything rightwards by however much the user wants to risk?

**Comment by Minoru** [comment]
@Friptick, the problem with introducing new settings is that it's nigh impossible to take them out.

**Comment by Friptick** [comment]
Fair enough. Your project, your call about how or if to proceed. I can only second what OP describes so succinctly:
>I believe one of the main selling points of feed readers is their distraction-free nature and comfort of use. In my experience, having text centered with narrow width limit in maximized window is the most enjoyable way to read. Currently I was able to only limit the text width.

**Comment by user451421541757324** [comment]
Any news about this? @Minoru

**Comment by Minoru** [comment]
No. Please see [my earlier comment here](https://github.com/newsboat/newsboat/issues/1320#issuecomment-1065886807) for an explanation on how we operate.
